forte/data/data_store.py

```diff
@@ -247,6 +247,23 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         entry += len(self._type_attributes[type_name]) * [None]
         return entry
 
+    @staticmethod
+    def _is_subclass(type_name: str, cls) -> bool:
+        r"""This function determines if a string class_type is
+        the subclass of a class.
+
+        Args:
+            type_name: A fully qualified name of an entry class
+            cls: A class.
+
+        Returns:
+            A boolean value whether this type belongs to a clc class
+            type or not.
+
+        """
+        entry_class = get_class(type_name)
+        return cls == entry_class.__base__
```

Comment:
Should also consider the case when `cls` is the grandparent of `entry_class`.

Reply:
OK， Got it.